src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -18,9 +23,28 @@ public OperatorCommandMap() {}
     @Inject
     public void setupMyCommands(
             OperatorInterface operatorInterface,
-            SetRobotHeadingCommand resetHeading)
-    {
+            SetRobotHeadingCommand resetHeading,
+            DriveSubsystem drive) {
         resetHeading.setHeadingToApply(90);
         operatorInterface.gamepad.getifAvailable(1).onTrue(resetHeading);
+
+        // Stolen from 2024
```

Comment:
This looks game specific. Should we remove it?

Reply:
Will remove after testing the features :)